update.sh

```diff
@@ -84,10 +92,10 @@ type -P snap &>/dev/null && sudo snap refresh
 type -P flatpak &> /dev/null && flatpak update
 
 # If Pacstall has been enabled
-if [[ -f "$HOME/.rhino/config/pacstall" ]]; then
+if [[ -f "/usr/share/rhino/config/pacstall" ]]; then
   # Install Pacstall
-  mkdir -p ~/rhinoupdate/pacstall/
-  cd ~/rhinoupdate/pacstall/
+  mkdir -p /usr/share/rhino/rhinoupdate/pacstall/
+  cd /usr/share/rhino/rhinoupdate/pacstall/
   wget -q --show-progress --progress=bar:force https://github.com/pacstall/pacstall/releases/download/1.7.3/pacstall-1.7.3.deb
```

Comment:
Requires sudo? Also update to latest version of script which uses 2.0.1.deb

Reply:
This is already in rhino-update. I just had to `git pull` first.